web/src/less/inc.var.less

```diff
@@ -63,3 +63,31 @@ code {
     }
   }
 }
+
+.cyclone-layout-header {
```

Comment:
duplicate?

Reply:
duplicate from index.less, keep the index.less  only for introducing other files